person.rb

```diff
@@ -3,7 +3,7 @@
 require_relative 'trimmer_decorator'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] rentals should be read and modify in the Person's object that will be created :+1:   `attr_accessor :rentals`